src/util/services_utils.js

```diff
@@ -0,0 +1,23 @@
+/* Copyright (C) 2024 NooBaa */
+'use strict';
+
+const signature_utils = require('./signature_utils');
+const { RpcError } = require('../rpc');
+
+const sdk_type_enum = {
+    OBJECT_SDK: 'object_sdk',
+    STS_SDK: 'sts_sdk',
+    ACCOUNT_SDK: 'account_sdk',
```

Comment:
I would avoid this enum if not needed and just send the actual sdk object to authenticate_request_by_service

Reply:
Changed, thanks.